src/__tests__/mode.ts

```diff
@@ -14,7 +14,7 @@ const defaultVariant: Variant = Variant.DEFAULT
 const defaultExternal: string = 'NONE'
 
 // define session
-const session = ace.createEditSession('', DefaultMode)
+const session = ace.createEditSession('', new DefaultMode.Mode())
```

Comment:
what does this change do, doesn't seem to be building?

Reply:
There's a TS type error otherwise, looking into it

Reply:
i'd be happy to approve the other two upgrades if you want to get those through first